go/vt/vttablet/onlineddl/executor.go

```diff
@@ -2065,7 +2069,7 @@ func (e *Executor) executeMigration(ctx context.Context, onlineDDL *schema.Onlin
 	// migration context and DDL as a previous one. We are only interested in our scenario in a duplicate
 	// whose predecessor is "complete". If this is the case, then we can mark our own migration as
 	// implicitly "complete", too.
-	if onlineDDL.RequestContext != "" {
+	{
```

Comment:
This is actually unrelated to the essence of the PR but something I cleaned up while I was here